### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Markkuuss**
Added an example of a textual configuration using the new MQTT plugin from Openhab 2.4 with the internal broker.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -225,6 +225,34 @@ to integrate with [ThingsBoard.io](https://thingsboard.io/):
...
+In the following is an example of a textual configuration using the new MQTT plugin from Openhab 2.4 with the internal broker.
+
+### Thing file
+```
+Bridge mqtt:systemBroker:MqttBroker "MQTT Broker" [ brokerid="embedded-mqtt-broker" ]
+{
+    Thing topic FicusBenjamin "Ficus Benjamin"
+    {
+        Channels:
+            Type number : light         "Lichtstärke"           [ stateTopic="miflora/FicusBenjamin", transformationPattern="JSONPATH:$.light" ]
```
Comment (hakan42) [inline README.md]: Could you please use english names so the international community has an easier way of understanding your examples?

Also check the other names in your example.

**Comment by ThomDietrich** [comment]
Hey @Markkuuss,
there are 1-2 spelling errors. Could you please go through once more? Thanks!

**Comment by Markkuuss** [comment]
I corrected the spelling mistake I found and I also adjusted the formatting. I hope it's okay now?

**Comment by ThomDietrich** [comment]
I've moved the section one level up. Thanks for the important addition!

**Pull request merged**